src/Sarif.UnitTests/TestData/SarifCurrentToVersionOneVisitor/v1/OneRunWithFiles.sarif

```diff
@@ -27,7 +27,7 @@
           }
         },
         "file:///home/buildAgent/bin/app.zip#/docs/intro.docx": {
-          "uri": "file:///docs/intro.docx",
```

Comment:
>file [](start = 18, length = 4)

I fixed this existing bug in the transformer.

Reply:
yes good catch, can you please add another test to this file that nests a second zip in this one, and another file object that is a nested file within that nested zip? this guarantees completeness in nested files testing.

---
In reply to: [247374942](https://github.com/Microsoft/sarif-sdk/pull/1211#discussion_r247374942) [](ancestors = 247374942)

Reply:
Done.

---
In reply to: [247377083](https://github.com/Microsoft/sarif-sdk/pull/1211#discussion_r247377083) [](ancestors = 247377083,247374942)